module/actor/actor-sheet.js

```diff
@@ -336,15 +336,19 @@ export class yzecoriolisActorSheet extends ActorSheet {
     // for display purposes we'll halve everything so that encumbrance makes
     // sense to users that are familiar with the rules.
     let enc = {
-      max: strengthValue / 2,
+      max: (strengthValue / 2) + this.actor.system.encumbranceMods,
       value: totalWeight / 2,
     };
     let pct = (enc.value / enc.max) * 100;
     if (enc.value === 0) {
       pct = 0;
     }
     enc.percentage = Math.min(pct, 100);
-    enc.encumbered = pct > 100;
+    if (pct > 100 || pct < 0) {
```

Comment:
Why the pct < 0?

Reply:
> Why the pct < 0?

With the modifiers you theroetically could get negative encumbrence but that wouldn't mark the enumbrance-bar as overloaded.

That fixxed that issue^^

Reply:
also could just always cap the min-encumbrance with 0 but I think than you could "oversee" some stuff like with "just 0" you could think "yeah I'm weak but no problems, we got this" but in reality you have injuries who says you have no arms or shoulders (bad example but I hope you get my idea behind it)

Reply:
There is an edge case when max = Strength * 2 - mod = 0... if your encumbrance is 0, the result is NaN otherwise it is Infinity. The code seems to work fine in both cases fine but that is error prone I think.

I think a negative max encumbrance is counter intuitive. So I think I'd cap max encumbrance to 0. The question is if max = 0 and you have nothing on you (enc = 0), are you encumbered of not? 

So maybe
- if enc = 0, pct = 0, encumbered = false (if you have nothing, you are never encumbered)
- else if max enc = 0, pct = 100, encumbered = true (if you have something and max enc 0, then you are encumbered)
- else ... normal computation

Would that make sense?

Reply:
> There is an edge case when max = Strength * 2 - mod = 0... if your encumbrance is 0, the result is NaN otherwise it is Infinity. The code seems to work fine in both cases but that is error prone I think.

Is that? In my installation it shows it like that:
![image](https://github.com/winks-vtt/yze-coriolis/assets/36819852/48edec93-d0f7-42cb-8c5d-0a1326cfe07b)
![image](https://github.com/winks-vtt/yze-coriolis/assets/36819852/306acc5b-0199-421d-9813-1e75bd20d544)
![image](https://github.com/winks-vtt/yze-coriolis/assets/36819852/f3421513-a886-43af-9fa0-1af60fdc5978)

Seems fine to me.

Reply:
Yeah, it looks fine. The max operation saves the day (and swallow Infinity or NaN). I'm use to code blowing up when dividing by 0 but it seems javascript handles it differently.